workflows/wf_theiaprok_illumina_se.wdl

```diff
@@ -424,7 +424,15 @@ workflow theiaprok_illumina_se {
             agrvate_agr_multiple = merlin_magic.agrvate_agr_multiple,
             agrvate_agr_num_frameshifts = merlin_magic.agrvate_agr_num_frameshifts,
             agrvate_version = merlin_magic.agrvate_version,
-            agrvate_docker = merlin_magic.agrvate_docker
+            agrvate_docker = merlin_magic.agrvate_docker,
+            srst2_vibrio_tsv = merlin_magic.srst2_vibrio_tsv,
+            srst2_vibrio_detailed_tsv = merlin_magic.srst2_vibrio_detailed_tsv,
+            srst2_vibrio_version = merlin_magic.srst2_vibrio_version,
+            srst2_vibrio_ctxA = merlin_magic.srst2_vibrio_ctxA,
+            srst2_vibrio_ompW = merlin_magic.srst2_vibrio_ompW,
+            srst2_vibrio_tcpA_ElTor = merlin_magic.srst2_vibrio_tcpA_ElTor,
+            srst2_vibrio_toxR = merlin_magic.srst2_vibrio_toxR,
+            srst2_vibrio_wbeN_O1 = merlin_magic.srst2_vibrio_wbeN_O1
```

Comment:
`srst2_vibrio_tcpA_ElTor` change to `srst2_vibrio_biotype`

and

`srst2_vibrio_wbeN_O1` change to `srst2_vibrio_serotype`